### ISSUE TITLE
Admob Ads

### ISSUE THREAD

**Issue by suman1985dutta** [naman14/Timber#399]
Why I am unable to integrate Admob banner Ads here? Please anyone help me....

**Comment by anburocky3** [comment]
Yes, you can integrate Admobs into Timber. But Make sure you follow the GNU License. (Y) Happy Coding.

**Comment by khursheduser** [comment]
https://en.m.wikipedia.org/wiki/GNU_General_Public_License

вт, 28 авг. 2018 г., 1:23 suman1985dutta <notifications@github.com>:

**Comment by anburocky3** [comment]
It may help you. https://tldrlegal.com/license/gnu-general-public-license-v3-(gpl-3)

**Comment by Debjit733133** [comment]
How to integrate admob ads in Timber music player.....   ????????
Please anyone help me...

**Comment by Debjit733133** [comment]
How to integrate admob ads in Timber music player.....   ????????
Please anyone help me...
